Presentation.Web/app/components/it-contract/tabs/it-contract-tab-reference.ts

```diff
@@ -64,7 +64,7 @@
                         field: "title",
                         title: "Dokumenttitel test",
                         template: data => {
-                            if (Kitos.Utility.Validation.validateUrl(data.url)) {
+                            if (Kitos.Utility.Validation.isValidExternalReference(data.url)) {
```

Comment:
hvad med overbliksbillederne for project og contract? Bliver den ikke brug der?

Reply:
Kan se at Project samt contract overview bruger ikke urlvalidation.

Reply:
ok, det burde de måske ligesom de andre overbliksbilleder nu hvor vi alligevel har fat i det koncept

Reply:
Overbliksbillederne er reference allerede valideret på grund af.
![image](https://user-images.githubusercontent.com/51359645/79343995-7f0d7400-7f2f-11ea-9cb6-3268a0b8a2de.png)

Reply:
ja og det kan man jo så også sige med de overbliksbilleder der rent faktisk bruger den, men årsagen til at du skal lave det i kontrakt og projekt er:
- Hver gang vi skal rette i en generel regel er det nemmere når vi kun skal rette den regel ét sted
- Overbliksbilledets data kan godt være ude af sync med den "gældende regel" så hvis man pludselig laver datakonverteringer eller lignende, så er det kun "referencer" fanen der gennemtvinger visningsreglen og det er forkert. 

**Så** lad os nu gøre det konsekven. Ellers bliver det "tilfældigt" om vi rammer rigtigt næste gang vi retter i reglen.

Reply:
@mbkstrongminds Se ☝️